conf/janus.eventhandler.gelfevh.jcfg.sample

```diff
@@ -0,0 +1,21 @@
+# This configures the gelf event handler. Appending necessary headers 
+# and sending messages via TCP or UDP (maybe quic soon :D)
```

Comment:
I missed this part here: `gelf` should be `GELF`. I understand it's cute, but the QUIC reference and the smiley should be removed :wink:

Reply:
This was marked as resolved, but it looks it wasn't?

Reply:
done, i missed this one completely  and not sure how